### PR TITLE
[TR] Add `Legal/Privacy`

### PR DESCRIPTION
**Pull request by zeusminus**
Add Turkish locale for `Legal/Privacy` article.

### CODE REVIEW

File: wiki/Legal/Privacy/tr.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- Avatarınız ve profil kapak resimleriniz
+- İmzanız
+
+Üstte belirtilen tüm alanlar herkese açık bir şekilde görüntülenebilir fakat bu alanlar aynı zamanda [ayarlar sayfasından](https://osu.ppy.sh/home/account/edit) anında ve kalıcı olarak geri çekilebilir.
+
+### Kullanıcı tarafından gönderilen içerikleri karşıya yüklerken
+
+Forumlarda gönderi oluşturduğunuzda, sohbete dahil olduğunuzda veya hizmetlerimize beatmap gibi bir içerik yüklediğinizde, gönderilen tüm içerikleri apaçık bir şekilde herkese açık hale getirmiş olursunuz. Çoğu durumda, bu içerikleri yükleme sonrasında dilediğiniz takdirde düzenleyebilir ve silebilirsiniz, ancak bazı durumlarda bu işlevsellik alaka düzeyini korumak ve kullanıcı tabanımıza sağladığımız hizmeti devam ettirmek adına kilitli olabilir.
+
+Örneğin, yüklediğiniz bir beatmao "dereceli" olursa, bu beatmap kullanıcı tabanının büyük bir kısmının puan alması için temel oluşturur. Bu noktada yüklenen içeriği silme seçeneği kaldırılacaktır.
```
Comment (eAkhaten) [inline wiki/Legal/Privacy/tr.md:53]:
```suggestion
Örneğin, yüklediğiniz bir beatmap "dereceli" olursa, bu beatmap kullanıcı tabanının büyük bir kısmının puan alması için temel oluşturur. Bu noktada yüklenen içeriği silme seçeneği kaldırılacaktır.
```

File: wiki/Legal/Privacy/tr.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+### Oyun istemcisine giriş yaparken
+
+osu! oyun istemcisi üzerinden hizmetimize bağlanırken, istemciye özel bir karakter dizisi mevcut oyun ortamınızı tanımlayabilmemize yardımcı olması için bize gönderilir. Bu, donanım ve yazılım yapılandırma bilgilerinizden elde edilen tanımlayıcıların bir kombinasyonu baz alınarak oluşturulur ve kişisel olarak tanımlayıcı bir bilgi içermeyecek düzeyde şifrelenir, ancak hizmetimize girişler arasındaki erişimlerinizi takip etmek için kullanılabilir.
```
Comment (eAkhaten) [inline wiki/Legal/Privacy/tr.md:57]:
```suggestion
osu! oyun istemcisi üzerinden hizmetimize bağlanırken, istemciye özel bir karakter dizisi, mevcut oyun ortamınızı tanımlayabilmemize yardımcı olması için bize gönderilir. Bu, donanım ve yazılım yapılandırma bilgilerinizden elde edilen tanımlayıcıların bir kombinasyonu baz alınarak oluşturulur ve kişisel olarak tanımlayıcı bir bilgi içermeyecek düzeyde şifrelenir, ancak hizmetimize girişler arasındaki erişimlerinizi takip etmek için kullanılabilir.
```

File: wiki/Legal/Privacy/tr.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+Bunun temel amacı sıralama sisteminin adil kalmasını sağlamak ve riskli bir konumdan hesabınıza erişildiği takdirde hesabınızı güvence altına almamıza yardımcı olmaktır. Bu, gizli kabul edilir ve yalnızca gerekli görüldüğü sürecek saklanır. Bu, aynı zamanda transfer edilemez, ve osu! ekosistemi dışında herhangi bir anlam ifade etmez.
```
Comment (eAkhaten) [inline wiki/Legal/Privacy/tr.md:59]:
```suggestion
Bunun temel amacı sıralama sisteminin adil kalmasını sağlamak ve riskli bir konumdan hesabınıza erişildiği takdirde hesabınızı güvence altına almamıza yardımcı olmaktır. Bu, gizli kabul edilir ve yalnızca gerekli görüldüğü sürece saklanır. Bu, aynı zamanda transfer edilemez, ve osu! ekosistemi dışında herhangi bir anlam ifade etmez.
```

**Review by eAkhaten — DISMISSED** [review]
good

**Review by eAkhaten — DISMISSED** [review]
good

File: wiki/Legal/Privacy/tr.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
+---
+legal: true
+---
+
+# osu! gizlilik politikası
+
+En son 16 Nisan 2021 tarihinde güncellendi. [Geçmişi buradan görüntüleyin](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/tr.md)
```
Comment (TicClick) [inline wiki/Legal/Privacy/tr.md:7]: please refer to the original page (`en.md`)

**Review by TicClick — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**